### PR TITLE
Alertmessage : Alert pop up message from antd added #360 done

### PR DESCRIPTION
**Pull request by VinayLodhi1712**
Alertmessage : Alert pop up message from antd added #360 done.

https://github.com/user-attachments/assets/c905016c-ee8d-47b9-b1da-ed006637f669

## Release Notes

- **New Features**
	- Enhanced user feedback in the Login and Signup components with Ant Design's messaging system for success and error notifications.
	- Detailed success and error messages now displayed with improved styling.

- **Bug Fixes**
	- Corrected a typo in the error handling middleware import.

- **Dependencies**
	- Updated Ant Design dependency to version 5.21.5 for improved functionality.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ require("dotenv").config();
 const cors = require("cors");
 const mongoose = require("mongoose");
 const logger = require("./config/logger");
-const errorMiddleware = require("./middlewares/errorMiddleware"); // Corrected typo
+const errorMiddleware = require("./middlewares/errrorMiddleware"); // Corrected typo
```
Comment (RamakrushnaBiswal) [inline backend/index.js:6]: @VinayLodhi1712 correct the speling

File: frontend/src/components/Pages/Login.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+
+  /* Success message line animation */
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Pages/Login.css:2]: don't use css files use tailwinf instead @VinayLodhi1712

File: frontend/src/components/Pages/Login.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 import photo from '../../assets/login.png';
 import { Link, useNavigate } from 'react-router-dom';
-import { message } from 'antd';
+import { message } from 'antd'; // Ant Design message component
 import Cookies from 'js-cookie';
 import { FaEye } from 'react-icons/fa';
 import { FaEyeSlash } from 'react-icons/fa6';
-
+import "./Login.css";
 const Login = () => {
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Pages/Login.jsx:9]: @VinayLodhi1712 here alsp

**Review by RamakrushnaBiswal — CHANGES_REQUESTED** [review]

**Comment by VinayLodhi1712** [comment]
@RamakrushnaBiswal Sir now you can check plz.

**Review by RamakrushnaBiswal — APPROVED** [review]

**Pull request merged**